src/API.jl

```diff
@@ -3,13 +3,6 @@
 export productbelief
 
 
-# MAYBE deprecate name
-manikde!( M::MB.AbstractManifold,
-          vecP::AbstractVector{P},
-          bw::Union{<:AbstractVector{<:Real},Nothing}=nothing;
-          partial::Union{Nothing, <:AbstractVector{<:Integer}}=nothing ) where P = ManifoldKernelDensity(M, vecP, vecP[1], bw=bw, partial=partial) 
```

Comment:
Hi @dehann, it looks like the removal of this function breaks IIF

Reply:
ah, i got the tests passing in IIF by merging the incomplete JuliaRobotics/IncrementalInference.jl#1309